application/common/models/PasswordHistory.php

```diff
@@ -58,6 +58,11 @@ private function hasAlreadyBeenUsedWithinLimit(): bool
         return false;
     }
 
+    public function expires(): string
+    {
+        return MySqlDateTime::format(strtotime('+1 year', strtotime($this->created_utc)));
```

Comment:
certainly, we discussed that option and thought it was prudent at this point to hold off until there was request for it.  If you like, I can add it now though.